supported-version/src/versions/magento-open-source/individual.json

```
@@ -739,6 +781,34 @@
         "nginx": "nginx:1.26",
         "os": "ubuntu-latest",
         "release": "2025-02-11T00:00:00+0000",
+        "eol": "2025-04-07T00:00:00+0000"
+    },
+    "magento/project-community-edition:2.4.7-p5": {
+        "magento": "magento/project-community-edition:2.4.7-p5",
+        "php": 8.3,
+        "composer": "2.7.4",
+        "mysql": "mariadb:10.6",
+        "elasticsearch": "elasticsearch:8.11.4",
+        "rabbitmq": "rabbitmq:3.13-management",
+        "redis": "redis:7.2",
+        "varnish": "varnish:7.5",
+        "nginx": "nginx:1.26",
+        "os": "ubuntu-latest",
+        "release": "2025-04-08T00:00:00+0000",
         "eol": "2027-04-09T00:00:00+0000"
+    },
+    "magento/project-community-edition:2.4.8": {
+        "magento": "magento/project-community-edition:2.4.8",
+        "php": 8.4,
+        "composer": "2.8.8",
+        "mysql": "mariadb:11.4",
+        "elasticsearch": "opensearch:2.19.1",
```

```suggestion
        "elasticsearch": "elasticsearch:8.17.4",
```

I don't like doing this. I think we need a separate key for elastic/open and redis/valkey.